eng/versioning/version_client.txt

```diff
@@ -32,7 +32,7 @@ com.azure:azure-cosmos-encryption;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-data-appconfiguration;1.1.5;1.1.6
 com.azure:azure-data-schemaregistry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
-com.azure:azure-data-tables;12.0.0-beta.1;12.0.0-beta.2
+com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.2
```

Comment:
Are we releasing `12.0.0-beta.2` or `12.0.0-beta.3`? I'm a bit confused on the change here as I was expecting the `current` version to be changed, not the `dependency` version.

Reply:
There is no stable version yet, so I assumed the dependency version should be incremented to the current version. Though currently no library depends on it so it doesn't seem like it would matter.

Also I assume that after the release, the version bump PR will change the `current` version to 12.0.0-beta.3. Right?